<launch>
    <node pkg="visual_servoing" type="visual_servoing" name="visual_servoing" output="screen">
        <param name="depth_image_topic" type="str" value="/mynteye/depth/image_raw" />
        <param name="image_topic" type="str" value="/mynteye/left/image_rect" />
        <param name="camera_info_topic" type="str" value="/mynteye/left/camera_info" />
        <param name="target_frame" type="str" value="world" />
        <param name="pose_topic" type="str" value="/rovio/odometry" />
        <param name="cutoff" type="bool" value="false" />
        <param name="debug" type="bool" value="true" />
        <param name="match_type" type="int" value="0" />
        <param name="patch_size" type="int" value="3" />
        <param name="simulation" type="bool" value="false" />
        <!-- <param name="camera_frame" type="str" value="/mynteye_left_mono_frame" /> -->
         <param name="camera_frame" type="str" value="/mynteye/left/image_raw" />
        <!-- <param name="drone_frame" type="str" value="/abs_correction" /> -->
        <param name="drone_frame" type="str" value="world" />
    </node>
    <node name="image_proc" pkg="image_proc" type="image_proc" ns="mynteye/left"/>
</launch>